app/frontend/src/components/forms/SubmissionsTable.vue

```diff
@@ -90,6 +90,9 @@ export default {
   },
   methods: {
     ...mapActions('form', ['fetchForm', 'fetchSubmissions']),
+    formatDate(date) {
+      return date ? new Date(date).toLocaleString() : 'N/A';
+    },
```

Comment:
This should use the `| formatDate` filter so there don't have to be date formatter methods down in each component.

Reply:
done